integration_tests/accessibility/accessibility.spec.js

```diff
@@ -46,4 +41,16 @@ context('Accessibility (a11y) Checks', () => {
       cy.pa11y(pa11yArgs)
     })
   })
+
+  it('error state (using /issues-needs as the example)', () => {
+    cy.visit(`/persons/${nomsNumber}/recalls/${recallId}/issues-needs`)
+    cy.get('[id="vulnerabilityDiversity"]').click()
+    cy.get('[id="vulnerabilityDiversityDetail"]').clear()
+    cy.get('[id="arrestIssues"]').click()
```

Comment:
Just need to click the continue button after page load, to get all the errors rendered. Stub get recall with an empty recall to ensure the form renders as empty.
`cy.clickLink('Continue')`